acceptance-tests/src/test/java/uk/gov/di/test/step_definitions/Login.java

```diff
@@ -186,12 +187,42 @@ public void theExistingUserRequestsThePhoneOtpCodeTimes(int timesCodeIncorrect)
         }
     }
 
+    @When("the existing user enters the phone otp code {string} 6 times")
```

Comment:
Make this step more readable and remove the need for {string}. See other comment against feature file for better description of what i mean